src/order/sup_indep.lean

```diff
@@ -16,18 +16,16 @@ In distributive lattices, this is equivalent to being pairwise disjoint.
 
 ## TODO
 
-As we don't have `lattice_bot`, we're forced to pick between `bounded_lattice` (which assumes `top`
-unnecessarily) and `distrib_lattice_bot` (which assumes distributivity unnecessarily). For now we
-pick `distrib_lattice_bot` for it to apply to `finset α`, although this use is redundant with
-`set.pairwise_disjoint`.
+Some results still require distributivity of lattices, in particular, the ones that rely on
```

Comment:
You've fulfilled the TODO. The lemmas that still require distributivity can't do without. Feel free to move them to their own `distrib_lattice` section. I should've done that myself.